src/ansys/fluent/core/session.py

```diff
@@ -133,10 +136,23 @@ def _build_from_fluent_connection(
         if self._start_transcript:
             self.transcript.start()
 
+        if FluentVersion(self.scheme_eval.version) < FluentVersion.v252:
+            self._app_utilities = AppUtilitiesOld(self.scheme_eval)
+        else:
+            self._app_utilities_service = self._fluent_connection.create_grpc_service(
+                AppUtilitiesService, self._error_state
+            )
+            self._app_utilities = service_creator("app_utilities").create(
```

Comment:
I think this is already initialized within FluentConnection?

Reply:
@mkundu1 Right. Thank you.